content/blog/2020-08-04-osf.md

```diff
@@ -0,0 +1,86 @@
+
+### How are they different?
+
+Ultimately the two services are different because they are designed for different purposes: facilitating software development vs facilitating scientific research. While the line separating these tasks is often blurry (or non-existant), supporting research requires accommodating files of all shapes and sizes, whereas software (generally) comes in one flavor: plain text. Furthermore, GitHub is intended for software developers, an inherently technical userbase, whereas OSF is designed to accomodate users with a much wider range of computational expertise.
+
+One area where the manifestation of these differences can be seen is in how OSF handles file versions. While OSF automatically stores all previous versions of files as new ones are uploaded, it's not a true version control system in the sense of something like git, which intelligently stores only the differences between file versions, making it possible to easily compare versions and automatically detect conflicting changes. On OSF, the latest version of a file is simply the last uploaded version regardless of how or whether its content changed. However, what OSF's versioning feature lacks in power and options, it makes up for in accessibility---another key differentiator. Creating a new version on OSF simply requires dragging and dropping an updated file onto the project's file browser
+
+Version control is one feature common to both GitHub and OSF, but let's look at some of the key features that are unique OSF that are purpose built for managing and disseminating research.
+
+#### Integrations
+
+First and foremost, I want to point out that adopting (or just testing out) OSF does not require upending your current workflows and abandoning services you're already happy with. That's because OSF directly integrates with many popular services through its [add-ons system][osf-addons]. So although OSF provides its own free cloud storage, if you're already using Dropbox, Google Drive, S3, etc, then you can simply add the relevent folder/bucket from that service to your OSF project. And if you're using GitHub/GitLab to manage your source code (good idea), then you can also link the relevant repository. Either way, they all show-up as first-class citizens in your OSF project, right alongside OSF's native cloud storage buckets.
```

Comment:
relevant
```suggestion
First and foremost, I want to point out that adopting (or just testing out) OSF does not require upending your current workflows and abandoning services you're already happy with. That's because OSF directly integrates with many popular services through its [add-ons system][osf-addons]. So although OSF provides its own free cloud storage, if you're already using Dropbox, Google Drive, S3, etc, then you can simply add the relevant folder/bucket from that service to your OSF project. And if you're using GitHub/GitLab to manage your source code (good idea), then you can also link the relevant repository. Either way, they all show-up as first-class citizens in your OSF project, right alongside OSF's native cloud storage buckets.
```